pkg/machinery/resources/cri/registry.go

```diff
@@ -0,0 +1,140 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package cri
+
+import (
+	"github.com/cosi-project/runtime/pkg/resource"
+	"github.com/cosi-project/runtime/pkg/resource/meta"
+	"github.com/cosi-project/runtime/pkg/resource/protobuf"
+	"github.com/cosi-project/runtime/pkg/resource/typed"
+	"github.com/siderolabs/crypto/x509"
+
+	config2 "github.com/siderolabs/talos/pkg/machinery/config/config"
+	"github.com/siderolabs/talos/pkg/machinery/config/types/v1alpha1"
+	"github.com/siderolabs/talos/pkg/machinery/proto"
+)
+
+// RegistriesSpecType is type of RegistriesSpec resource.
+const RegistriesSpecType = resource.Type("Registries.net.talos.dev")
```

Comment:
if we call it `registries`, we should name the constant `RegistriesType`, and cascading down.

I would probably name it `RegistriesConfig` though

Reply:
Done!